testsuite/MDAnalysisTests/utils/test_persistence.py

```diff
@@ -125,9 +121,8 @@ def test_unpickle_reanchor(self, universe, pickle_str, ag):
         # now it goes back into the anchor list again
         universe.make_anchor()
         newag = cPickle.loads(pickle_str)
-        assert_array_equal(ag.indices, newag.indices)
```

Comment:
does this PR also do all `assert_array_equal` or just the ones you saw while doing assert_?

Reply:
just that one. 

I think I can use the same technique to get rid of all `assert_array_equal` later today.